### PR TITLE
Product listing filter

### PR DESCRIPTION
**Pull request by abhishektanwar**
In this commit 
- Filter logic on product listing page has been added
- Displaying products of a category when a specific category is clicked from home page

Files to check:
- src/Contexts/ProductListingContext/index.jsx
- src/Reducers/productsReducer.js
- src/components/CategoryCard/index.jsx
- src/components/FilterBar/index.jsx
- src/components/FeaturedContainer/index.jsx
- src/components/SelectedFilter/index.jsx
- src/helpers/helpers.js
- src/screens/ProductListing/index.jsx

### CODE REVIEW

File: src/Reducers/dispatchActionTypes.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export const dispatchActionTypes={
+  LOAD_DATA: "LOAD_DATA",
+	LOADING_DATA: "LOADING_DATA",
+	PRICE_RANGE: "PRICE_RANGE",
+	HIGH_TO_LOW: "HIGH_TO_LOW",
+	LOW_TO_HIGH: "LOW_TO_HIGH",
+	CATEGORY:"CATEGORY",
```
Comment (srishtigautam21) [inline src/Reducers/dispatchActionTypes.js:7]: Didn't knew we could do something like this also

**Review by srishtigautam21 — APPROVED** [review]
Clean, understandable and readable code
Good to merge

File: src/App.js [hunk 2]
```diff
@@ -1,35 +1,34 @@
...
-            </nav>
-            <Routes>
-              <Route path="/" exact element={<Home />} />
-              <Route path="/products" exact element={<ProductListing />} />
-              <Route path="/wishlist" exact element={<Wishlist />} />
-              <Route path="/cart" exact element={<Cart />} />
-					    <Route path='/mock-api' element={<MockmanEs />} />
-            </Routes>
+            <ProductListingProvider>
+              {/* <Login /> */}
```
Comment (iRohitGaur) [inline src/App.js]: remove commented code

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

**Pull request merged**